frb/frb.py

```diff
@@ -185,6 +189,39 @@ def set_ee(self, a, b, theta, cl, stat=True):
         #
         return
 
+    @property
+    def sig_a(self):
+        """
+        Combined semi-major axis error
+
+        Returns:
+            float:
+
+        """
+
+        if len(self.eellipse) == 0:
+            return None
+        siga = self.eellipse['a']  # arcsec
+        if 'a_sys' in self.eellipse.keys():
+            siga = np.sqrt(self.eellipse['a_sys']**2 + siga**2)
```

Comment:
I think this assumes the two ellipses are aligned. i.e. there is no relative rotation between the two. It's not taking `theta` into account at all.

Reply:
agreed.  So far they have been aligned.